### DISCUSSION TITLE
Calculating the wavenumber

### DISCUSSION THREAD

**Discussion by sarriegi** [NOAA-EMC/WW3#1018] [Q&A]
Hello, I am writing a matlab code, where I use the spectrum output of WW3 to calculate the Stokes drift transport. Then I would like to compare the Stokes drift transport that I calculated, to the Stokes drift transport that is WW3's output and get the same result for both. Let's say that I have the spectrum in 5 grid points (whose depth is around 10-30 m). Then I would like to calculate the wavenumber in each one of the grid points, using the WW3's frequencies and the dispersion relationship. Do you know which value of depth should be used for the dispersion relationship? Should it be the depth for each one of these 5 grid points? When I use a constant value for all grid points around 20 m instead of the bathymetry value for each one of the grid points, then the Stokes drift transport value from my calculation is much close to the Stokes drift transport from WW3's output (still not identical however).

@aliabdolali 

Angelos

**Comment by aronroland** [comment]
the depth at the point of concern of course...
